requirements.txt

```diff
@@ -0,0 +1 @@
+Flask==1.0.2
```

Comment:
Don't you have more dependencies? Jinja? Protobuf? Please pip freeze in a virtualenv to check all. Also, run all the files and mention all the dependencies used.

Reply:
@shreybatra Jinja2 comes as an install requirement within Flask ( `Jinja2>=2.10` ).

Reply:
Resolved in 1d087f2